.eslintrc

```diff
@@ -27,8 +27,9 @@
   },
   "overrides": [
     {
-      "files": [ "**/*.js" ],
+      "files": ["**/*.js"],
       "rules": {
+        "@typescript-eslint/explicit-module-boundary-types": "off",
```

Comment:
I was writing some of the test fixtures using plain JS to avoid having to type them and ran into this rule. Happy to switch the test fixtures to TS, but I noticed the existing tests were already in JS, so it seems like that's how we're rolling here.

Reply:
@benweissmann is the expert here

Reply:
Yup, having the testing code in JS seems fine to me